components/portfolio/WithdrawButton.tsx

```diff
@@ -69,18 +75,21 @@ const createWithdrawExtrinsic = (
     "100000000000",
```

Comment:
is this ZTG? Will it vary from asset to asset?

Reply:
This is just to estimate fees. I don't think the amount transferred really has a effect, hardcoding it saves having to keep making an rpc call whenever the amount is changed